File: 20220913/ompp-0.8.5/opari/tool/Test/test5.c
#include <stdio.h>

int main() {
  int i, j, k, l;
  #pragma omp parallel         /* parallel */ \
      for                      /* do */ \
      lastprivate(k)           /* do */ \
      private(i,j)             /* parallel */ \
      lastprivate              /* do */ \
      (                        /* do */ \
        l                      /* do */ \
      ) schedule(dynamic       /* do */ \
      )
  for(i=0; i<4;++i) {
    printf("pdo %d\n", i);
  }
}

File: 20220913/ompp-0.8.5/opari/tool/Test/test5.f90
      program test5
      integer i, j, k, l
      !$omp parallel                          & !parallel
      !$omp & do                              & !do
      !$omp & lastprivate(k)                  & !do
      !$omp & private(i,j)                    & !parallel
      !$omp & lastprivate                     & !do
      !$omp & (                               & !do
      !$omp &   l                             & !do
      !$omp & ) schedule(dynamic              & !do
      !$omp   )
      do i=1,4
        write(*,*) "pdo", i
      enddo
      !$omp end parallel do
      end
